apps/concierge_site/lib/controllers/trip_controller.ex

```diff
@@ -17,10 +17,21 @@ defmodule ConciergeSite.TripController do
   @valid_facility_types ~w(bike_storage elevator escalator parking_area)
 
   def index(conn, _params, user, _claims) do
-    trips = Trip.get_trips_by_user(user.id)
-    render(conn, "index.html", trips: trips, user_id: user.id)
+    conn
+    |> communication_mode_flash(user)
+    |> render("index.html", trips: Trip.get_trips_by_user(user.id), user_id: user.id)
+  end
+
+  defp communication_mode_flash(conn, %User{communication_mode: "none"}) do
+    put_flash(
+      conn,
+      :warning,
+      "Alerts are disabled for your account. To resume alerts, check your account Settings."
```

Comment:
Would it be helpful if this included a link to your account settings?

Reply:
Currently flash content can only be plain text (it is HTML-escaped on render), so before making that possible we'd have to go through and ensure all flash content is HTML-safe. I might look into that anyway since I am also waiting on a content review.

Reply:
Ah right, I forgot about that. Well I'll leave it up to you, but it doesn't seem that important.